### PR TITLE
Reuse fixes

### PR DESCRIPTION
**Pull request by yegorich**
This patch series fixes some of the issues mentioned by the REUSE utility

### CODE REVIEW

File: LICENSES/GPL-2.0-only.txt [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+REDISTRIBUTE THE PROGRAM AS PERMITTED ABOVE, BE LIABLE TO YOU FOR DAMAGES,
+INCLUDING ANY GENERAL, SPECIAL, INCIDENTAL OR CONSEQUENTIAL DAMAGES ARISING
+OUT OF THE USE OR INABILITY TO USE THE PROGRAM (INCLUDING BUT NOT LIMITED
+TO LOSS OF DATA OR DATA BEING RENDERED INACCURATE OR LOSSES SUSTAINED BY
+YOU OR THIRD PARTIES OR A FAILURE OF THE PROGRAM TO OPERATE WITH ANY OTHER
+PROGRAMS), EVEN IF SUCH HOLDER OR OTHER PARTY HAS BEEN ADVISED OF THE
+POSSIBILITY OF SUCH DAMAGES.
+
+		     END OF TERMS AND CONDITIONS
+
```
Comment (hartkopp) [inline LICENSES/GPL-2.0-only.txt]: I was not aware of the following how-to.
Are you sure we would need to add that following text and not end the license file here?

Comment (yegorich) [reply LICENSES/GPL-2.0-only.txt]: This is the end the license file IMHO. This approach is suggested by REUSE project and Linux kernel also follows this structure.

Comment (hartkopp) [reply LICENSES/GPL-2.0-only.txt]: Ok. I wasn't aware that this how-to is really part of the license text.
Strange but hey ... then it's obviously the right thing to do.
Acked-by: Oliver Hartkopp <socketcan@hartkopp.net>

**Comment by yegorich** [comment]
There is one more questio left. Linux kernel provides license files with additional fields (https://github.com/torvalds/linux/blob/master/LICENSES/preferred/BSD-3-Clause). So should we this approach or just put the license text only?

**Comment by yegorich** [comment]
v2: added Oliver's Ack and replaced BSD license with text only version as the other two.

**Pull request merged**